src/components/Sidebar.js

```diff
@@ -16,10 +17,19 @@ const SidebarContainer = styled.div`
   }
 `
 
-const Logo = styled.img`
+const Logo = styled.img.attrs(p => ({
+  src: p.theme.custom_imgs === 'hc' ? hc_logo : logo,
+}))`
   width: 80px;
   height: 85px;
   margin: 30px 0 0px 50px;
+
+  ${p =>
+    p.theme.custom_imgs === 'hc' &&
+    css`
```

Comment:
why do we need this?

Reply:
HC logo is more rectangle than square, so just fixing width and padding to make it look nicer

Reply:
You can't do that without `css`?

Reply:
how else should i do the conditional css bit?

Reply:
does this not work
```
  ${p =>
    p.theme.custom_imgs === 'hc' &&
    `
      width: 120px;
      margin: 30px 0 0px 60px;
    `}
`
```

Reply:
lemme try that